datacenterlight/templates/datacenterlight/landing_payment.html

```diff
@@ -1,190 +1,185 @@
+                        <hr>
+                        <p>{% trans "Disk space"%} <strong class="pull-right">{{vm.disk_size}}2 GB</strong></p>
+                        <hr>
+                        <p>{% trans "Configuration"%} <strong class="pull-right">{{request.session.template.name}}CentOS 7</strong></p>
+                        <hr>
+                        <p class="last-p"><strong>{%trans "Total" %}</strong>&nbsp;&nbsp;<small>({%trans "including VAT" %})</small> <strong class="pull-right">{{request.session.specs.price}}15 CHF/{% trans "Month" %}</strong></p>
                     </div>
                 </div>
             </div>
-        </div>
-        <div class="row">
-            <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 dcl-billing-sec">
-                <div class="col-xs-12 col-sm-5 col-md-6  billing dcl-billing">
-                    <h3><b>{%trans "Billing Address"%}</b></h3>
-                    <hr>
+            <div class="dcl-payment-box">
+                <div class="dcl-payment-section">
+                    {% if not request.user.is_authenticated %}
+                        <h3><b>{%trans "Sign up"%}</b></h3>
+                    {% else %}
+                        <h3><b>{%trans "Billing Address"%}</b></h3>
+                    {% endif %}
+                    <hr class="top-hr">
                     <form role="form" id="billing-form" method="post" action="" novalidate>
+                        {% if not request.user.is_authenticated %}
+                            <div class="form-group">
+                                <label class="sr-only" for="id_signup_name">{% trans "Name" %}</label>
+                                <input type="text" id="id_signup_name" class="form-control" name="signup_name" placeholder="{% trans 'Name' %}">
+                            </div>
+                            <div class="form-group">
+                                <label class="sr-only" for="id_signup_email">{% trans "Email Address" %}</label>
+                                <input type="email" id="id_signup_email" class="form-control" name="signup_email" placeholder="{% trans 'Email Address' %}">
+                            </div>
+                        {% endif %}
                         {% for field in form %}
                         {% csrf_token %}
                         {% bootstrap_field field show_label=False type='fields'%}
                         {% endfor %}
                     </form>
                 </div>
-                <div class="col-xs-12 col-sm-7 col-md-6 creditcard-box dcl-creditcard">
+            </div>
+            <div class="dcl-payment-box">
+                <div class="dcl-payment-section">
                     <h3><b>{%trans "Credit Card"%}</b></h3>
-                    <hr>
+                    <hr class="top-hr">
+                    <p>
+                        {% blocktrans %}
```

Comment:
For blocktrans, I suggest that we add text immediately after the tag, to avoid ugly spaces and new line characters in po file.
Example:
```
{% blocktrans %}Please fill in your credit card information below. We are using <a href="https://stripe.com" target="_blank">Stripe</a> for payment and do not store your information in our database.{% endblocktrans %}
```